geopandas/tests/test_plotting.py

```diff
@@ -347,13 +347,14 @@ def test_style_kwargs(self):
 
 
 class TestPySALPlotting:
-
     @classmethod
     def setup_class(cls):
         try:
             import pysal as ps
         except ImportError:
             raise pytest.skip("PySAL is not installed")
+        except SyntaxError:
+            raise pytest.skip("PySAL fails Python 2 syntax")
```

Comment:
yeah, this is a strange one, seems the package is broken on py2, reported it: https://github.com/conda-forge/pysal-feedstock/issues/8